README.md

```diff
@@ -39,10 +38,37 @@ You can find a list of available interfaces using
 
 `.\geneva-cli.exe list-adapters`
 
+## How to test
+
+You can test or rather verify that the intercept mode is running by simply running it with a validated strategy. The output should show that packets are being rerouted. You can monitor your adapter in [WireShark](https://www.wireshark.org/) and you should notice a huge uptick in packets being sent from your adapter.
```

Comment:
The only strategy I've been able to run is the one in s.txt.  Which other strategies did you test with?  Also many strategies generate error statements, but the reference is malformed and the error is `nil`.  Here's an example using `[TCP:flags:S]-duplicate(tamper{TCP:flags:replace:SA},)-| \/`, which is pulled straight from strategies.txt:

```
.\geneva-cli.exe intercept -interface "vEthernet (Virtual Switch - WiFi Adapter)" -strategyFile .\from-strategies.txt
I: 14:34:17 outbound \/ inbound {[] ["[TCP:flags:S]-duplicate(tamper{TCP:flags:replace:SA}(send,),)-|"]}
I: 14:34:17 starting up
I: 14:34:17 starting statistics thread
I: 14:34:17 intercepting traffic on vEthernet (Virtual Switch - WiFi Adapter) (idx 11)
I: 14:34:17 using filter "ip and !loopback and ifIdx == 11"
I: 14:34:17 opening handle to WinDivert
E: 14:34:22 failed to send packet after error applying strategy: %!w(<nil>)
I: 14:34:22 intercepted/injected packets: 67/67 inbound, 87/88 outbound; 1 errors
E: 14:34:22 failed to send packet after error applying strategy: %!w(<nil>)
... same line 14 more times ...
I: 14:34:26 stopping all threads
I: 14:34:26 closing WinDivert handle
I: 14:34:26 intercepted/injected packets: 820/820 inbound, 756/772 outbound; 16 errors
```

I was also unable to verify that the strategy specified in s.txt was actually applied as expected.  How did you verify this?

Reply:
The program also panics if the strategy file does not exist.

Reply:
So the code expects one strategy passed to it in a file, strategies.txt is supposed to be a reference of known useful strategies. I should make clear in the readme that passing strategies.txt to intercept is a bad idea.

For the panic, I will update the code to check for the file first and output an error.